src/main/kotlin/no/nav/syfo/aksessering/api/dataSykmeldingDTO.kt

```diff
@@ -68,7 +69,8 @@ data class BehandlingsutfallDTO(
 data class RegelinfoDTO(
     val messageForSender: String,
     val messageForUser: String,
-    val ruleName: String
+    val ruleName: String,
```

Comment:
Har vi denne informasjonen lagret også for eldre sykmeldinger?

Reply:
Nei sant det, det vil vi jo ikkje ha, setter den som valgfri

Reply:
Det gjelder kanskje de andre feltene også?

Reply:
nvm :P